app/src/main/java/com/shagalalab/constitution/ui/search/SearchResultFragment.kt

```diff
@@ -63,7 +69,54 @@ class SearchResultFragment : Fragment(R.layout.fragment_search_result) {
         })
     }
 
+    private var noResultMsg = ""
+    private var hyperlink = ""
```

Comment:
You don't need these two global (class wide) variables, you can pass the values as method arguments, if needed.